packages/std/src/traits.rs

```diff
@@ -67,16 +67,16 @@ pub trait Api: Copy + Clone + Send {
     fn human_address(&self, canonical: &CanonicalAddr) -> StdResult<HumanAddr>;
 }
 
-// QuerierResponse is a short-hand alias as this type is long to write
-pub type QuerierResponse = Result<ApiResult<Binary>, SystemError>;
+/// A short-hand alias for the two-level query result (1. accessing the contract, 2. executing query in the contract)
+pub type QuerierResult = SystemResult<ApiResult<Binary>>;
```

Comment:
This is simpler.
If we can make `ApiError` == `StdError`, then this would be super clean (but that is a different issue)

Reply:
Interesting thought. With https://github.com/CosmWasm/cosmwasm/issues/271 and https://serde.rs/field-attrs.html#skip this should be possible. I just wonder if it is possible to construct an empty `Backtrace` in deserialize.

Reply:
That's promising: https://github.com/CosmWasm/cosmwasm/issues/303